docs/kits/Business Partner Kit/Documentation BPDM/page_documentation-bpdm.md

```diff
@@ -0,0 +1,61 @@
+
+Overall, BPDM plays a crucial role in creating a standardized and secure ecosystem for the exchange of business partner data in the automotive industry. Its services enable Catena-X members to easily access reliable and up-to-date business partner data and to share their own data with confidence, while ensuring data security and privacy.
+
+## Getting Started
+
+Getting started is easy. Here is a summary of the information that might help you get started:
+
+You will need to get a technical user and authentication via OAuth 2.0. You can find the getting started documentation with a description on how to get the technical user here. Once you have technical user, then you can get secrets from [vault](https://centralidp.demo.catena-x.net/) and can be used for authentication.
+After that, you start an authorization request via the API and get an access token, with that you can start requesting data from the resource. API requests without authentication will fail.
+
+Authorization URL
+
+[https://centralidp.demo.catena-x.net/auth/realms/CXCentral/protocol/openid-connect/auth](https://centralidp.demo.catena-x.net/auth/realms/CXCentral/protocol/openid-connect/auth)
+
+Token URL
+
+[https://centralidp.demo.catena-x.net/auth/realms/CX-Central/protocol/openidconnect/token](https://centralidp.demo.catena-x.net/auth/realms/CX-Central/protocol/openidconnect/token)
+
+### BPDM Pool
+
+1. BPDM Pool is a SpringBoot Kotlin software project managed by Maven.
+2. The project can be run with the following command: mvn clean spring-boot:run
+3. Prerequisites: Maven, JDK17, PostgreSQL 14.2, OpenSearch 2.1.0, Keycloak 17.0.0, Connection to an SaaS.
+4. The project requires a Postgresql database and an Opensearch instance to be available to connect to.
+5. Default configuration: postgres to run on localhost on port 5432, Opensearch to run on localhost on port 9200.
+6. Configuration files: application.properties, application-auth.properties, and application-saas.properties.
+7. REST API documentation: <http://localhost:8080/api/swagger-ui>.
+8. Profiles: default, auth, and saas.
+
+### BPDM Gate
+
+1. BPDM Gate is a SpringBoot Kotlin software project managed by Maven.
+2. The project can be run with the following command: mvn clean spring-boot:run
+3. Prerequisites: Maven, JDK17, Connection to an SaaS, Connection to BPDM Pool API, Keycloak 17.0.0.
+4. The Gate requires a remote SaaS storage and datasource to exchange data with.
+5. Default configuration: BPDM Pool instance expected at localhost with port 8080.
+6. Configuration files: application.properties, application-auth.properties, and application-pool-auth.properties.
+7. REST API documentation: <http://localhost:8081/api/swagger-ui>.
+8. Profiles: default, auth, and auth-pool.
```

Comment:
The Kit here needs to distinguish more clearly between two different use cases: A person who wants to run the BPDM service (either locally or deployed somewhere) and a person who wants to integrate against a deployed BPDM API. The INSTALL.md covers a lot of the first use case. The second use case is not covered well in the repository and should be the focus here in the Kit. 

The first part talks about how someone can authenticate to the API and then here it goes on what is required to install/deploy the API. Better separate and focus clearly on the 'usage' use case.